src/Component/Symbolizer/IconEditor/IconEditor.tsx

```diff
@@ -187,14 +201,14 @@ export const IconEditor: React.FC<IconEditorProps> = (props) => {
               // we overwrite imageFieldProps with the props
               // that were explicitly set as props on IconEditor.
               {...imageFieldProps}
```

Comment:
These props can now probably be hardcoded here and then overwritten via composition in `ImageField`